staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/cbor_test.go

```diff
@@ -121,15 +121,18 @@ func (p *anyObject) UnmarshalCBOR(in []byte) error {
 	return modes.Decode.Unmarshal(in, &p.Value)
 }
 
-type structWithRawExtensionField struct {
-	Extension runtime.RawExtension `json:"extension"`
+type structWithRawFields struct {
+	FieldsV1            metav1.FieldsV1       `json:"f"`
```

Comment:
this fine, but I'd also like a test that uses an actual embedded ObjectMeta.  I don't expect a difference, but you never expect it.